test/elm/library/test.coffee

```diff
@@ -74,3 +74,30 @@ describe 'Using CommonLib2', ->
 
   it "should execute function from included library that uses expression", ->
     @exprSortsOnFunc.exec(@ctx).should.eql [{N: 1}, {N: 2}, {N: 3}, {N: 4}, {N: 5}]
+
+describe 'Using CommonLib and CommonLib2', ->
+  @beforeEach ->
+    setup @, data, [p1], {}, {}, new Repository(data)
+    @results = @executor.withLibrary(@lib).exec_patient_context(@patientSource)
+    @commonLocalIdObject = @results.localIdPatientResultsMap['1'].Common
+    @common2LocalIdObject = @results.localIdPatientResultsMap['1'].Common2
+
+  it "should contain TheParameter localId in the localIdMap", ->
+    theParameterLocalId = @lib.includes.common2.expressions.TheParameter.localId
+    @common2LocalIdObject[theParameterLocalId].should.exist
```

Comment:
I think this (and others like it) needs to be:

```coffee
should.exist(@common2LocalIdObject[theParameterLocalId])
```

If `@common2LocalIdObject[theParameterLocalId]` was `null` or `undefined`, then the test as it is now would actually throw because you're trying to access `.should` on something that's `null`/`undefined`.

See: http://shouldjs.github.io/#should-exist

Reply:
BTW... I wish I used _chai_ in this project.  Probably should switch at some point.  Oh well.

Reply:
Good catch, those checks are never actually checked because the `()` is missing that makes them actually get called.